_posts/2019-09-26-aws-waf-association-with-api-gateway.markdown

```diff
@@ -1,23 +1,21 @@
 ---
 layout: post
-title:  "AWS WAF association with Api Gateway"
+title:  "AWS WAF association with API Gateway"
 date:   2019-09-26
 tags: AWS API WAF
 author: Natalie Laing
 image: img/lock.png
 ---
 
-I was recently trying to attach a WAF regional ACL to an api Gateway using cloudformation and I ran 
-into problems when the api gateway was created using serverless.
-The majority of the docs that I found online outlined how to attach your WAF to an ELB, I'm unsure whether this is because support for attaching WAF to api gateway was announced in November 2018 where support for ELB's has been around since 2016.
+I was recently trying to attach a WAF (Web Application Firewall) regional ACL (Access Control List) to an API (Application Programming Interface) Gateway using cloudformation and I ran into problems when the API gateway was created using Serverless Framework.
```

Comment:
`API Gateway` is a product - don't define the acronym here.

`API gateway` should be capitalised.

Reply:
There are multiple instances - if you are referring to the AWS service, it should be capitalised.

If you're referring to the concept then it shouldn't

Adding a reference for further reading is useful, and means you don't have to explicitly define the term:
* https://whatis.techtarget.com/definition/API-gateway-application-programming-interface-gateway
* https://en.wikipedia.org/wiki/API_management